.gitignore

```diff
@@ -2,19 +2,22 @@ htsjdk.iws
 .command_tmp
 atlassian-ide-plugin.xml
 /htsjdk.version.properties
-/test-output
+/test-output/
 
 #intellij
 .idea/
 src/htsjdk.iml
 *.iml
 *.ipr
 *.iws
-out/
-
+/out/
 
+#gradle stuff
+/.gradle/
```

Comment:
Maybe we want to ignore .gradle folders if they appear anywhere in the subtree?  It's unclear...